[2025-10-07T10:17:16.953+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T10:17:17.019+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:16:29.686945+00:00 map_index=12 [queued]>
[2025-10-07T10:17:17.049+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:16:29.686945+00:00 map_index=12 [queued]>
[2025-10-07T10:17:17.056+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-10-07T10:17:17.085+0000] {taskinstance.py:2890} INFO - Executing <Mapped(_PythonDecoratedOperator): process_batch> on 2025-10-07 10:16:29.686945+00:00
[2025-10-07T10:17:17.106+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=922) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T10:17:17.112+0000] {standard_task_runner.py:72} INFO - Started process 1090 to run task
[2025-10-07T10:17:17.118+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'vnstock_weekly_etl_incremental', 'process_batch', 'manual__2025-10-07T10:16:29.686945+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp4cmsl_d5', '--map-index', '12']
[2025-10-07T10:17:17.124+0000] {standard_task_runner.py:105} INFO - Job 58: Subtask process_batch
[2025-10-07T10:17:17.168+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-07T10:17:17.722+0000] {task_command.py:467} INFO - Running <TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:16:29.686945+00:00 map_index=12 [running]> on host e3278e0f0bbc
[2025-10-07T10:17:18.195+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='vnstock_weekly_etl_incremental' AIRFLOW_CTX_TASK_ID='process_batch' AIRFLOW_CTX_EXECUTION_DATE='2025-10-07T10:16:29.686945+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-07T10:16:29.686945+00:00'
[2025-10-07T10:17:18.205+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-10-07T10:17:18.209+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.217+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.217+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.218+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.219+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVO: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.219+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.220+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.221+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.226+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.227+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.228+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.228+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.240+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.240+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.241+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.242+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PV2: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.242+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTX: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.243+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.243+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.244+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.244+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.249+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTO: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.249+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.251+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.252+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.253+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.253+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.254+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.254+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.255+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.261+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.269+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.270+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSW: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.271+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.277+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.277+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.278+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.278+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.279+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.279+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.280+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.280+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.281+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.281+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PRT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.282+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PRO: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.283+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PRE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.283+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PRC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.284+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PQN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.288+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PPY: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.293+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PPT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:18.293+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T10:17:18.318+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-10-07T10:17:18.321+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=vnstock_weekly_etl_incremental, task_id=process_batch, run_id=manual__2025-10-07T10:16:29.686945+00:00, map_index=12, execution_date=20251007T101629, start_date=20251007T101717, end_date=20251007T101718
[2025-10-07T10:17:19.468+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T10:17:21.427+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T10:17:21.445+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
